invisible_cities/evm/pmaps_test.py

```diff
@@ -43,6 +44,13 @@ def s2si_input(draw, min_size=1, max_size=10):
     return size, t, E, qs1, qs2
 
 
+def rms(x, y):
+    if len(x) == len(y) < 2: return 0
+    mean = np.average(x, weights=y)
+    var  = np.average((x-mean)**2, weights=y)
+    return var**0.5
+
```

Comment:
Can we not use `np.var`? (also in `xy-algorithms`?)

Also, since we do this in a couple places, maybe we can put an rms function in `invisible_cities.core`

Reply:
`np.var` does not accept weights, that is why we use this `np.average` formulation.